deploy/helm/edot-collector/kube-stack/values.yaml

```diff
@@ -0,0 +1,943 @@
+# For advanced configuration options, refer to the [official OpenTelemetry Helm chart](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-kube-stack/values.yaml)
```

Comment:
Now that this file resides in `elastic-agent` repo, I think we should link the users to our `elastic/opentelemetry` README where we have the information about the most common configuration settings that the users should apply.

```suggestion
# For installation and configuration options, refer to the [installation instructions](https://github.com/elastic/opentelemetry/blob/main/docs/kubernetes/operator/README.md)

# For advanced configuration options, refer to the [official OpenTelemetry Helm chart](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-kube-stack/values.yaml)
```